src/building/bootstrapping.md

```diff
@@ -57,13 +57,13 @@ When you use the bootstrap system, you'll call it through `x.py`.
 However, most of the code lives in `src/bootstrap`.
 `bootstrap` has a difficult problem: it is written in Rust, but yet it is run
 before the rust compiler is built! To work around this, there are two
-components of bootstrap: the main one written in rust, and `bootstrap.py`.
+components of bootstrap: the main one written in Rust, called `rustbuild`,
+and `bootstrap.py`.
 `bootstrap.py` is what gets run by x.py. It takes care of downloading the
-`stage0` compiler, which will then build the bootstrap binary written in
-Rust.
+bootstrap compiler, which will then compile `rustbuild`.
```

Comment:
What do you think?

```suggestion
bootstrap compiler (a beta version of `rustc`), which will then compile `rustbuild`.
```

Reply:
The bootstrap compiler is not necessarily the beta compiler: https://rustc-dev-guide.rust-lang.org/building/bootstrapping.html?highlight=usually,beta#stages-of-bootstrapping

Reply:
How about this then:

```suggestion
bootstrap compiler (usually the latest beta version of `rustc`), which will then compile `rustbuild`.
```

Reply:
Is there a reason to say this twice? The bootstrap compiler is already discussed at the top of the page under 'stage 0'

Reply:
I guess not. Feel free to mark as resolved :)